tequilapi/endpoints/connection.go

```diff
@@ -75,19 +95,33 @@ func NewConnectionEndpoint(manager connection.Manager, ipResolver ip.Resolver, s
 }
 
 // Status returns status of connection
+// swagger:operation GET /connection Connection connectionStatus
+// ---
+// summary: Returns connection status
+// description: Returns status of current connection
+// responses:
+//   200:
+//     description: Status
+//     schema:
+//       "$ref": "#/definitions/statusResponse"
+//   500:
+//     description: Internal server error
+//     schema:
+//       "$ref": "#/definitions/errorMessage"
 func (ce *ConnectionEndpoint) Status(resp http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
 	statusResponse := toStatusResponse(ce.manager.Status())
 	utils.WriteAsJSON(statusResponse, resp)
 }
 
+// Creates starts new connection
```

Comment:
should be "Create", shouldn't it be caught by linter? Did you run it?

Reply:
fixed, linter configured to have some tolerance